### DISCUSSION TITLE
Cannot get "Galaxy Brain" achievement

### DISCUSSION THREAD

**Discussion by S1i** [community/community#45578] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've been searching and trying to get the Galaxy Brain achievement for a while now, I read somewhere you can not have marked your own comments to your own questions as the answer. I have done that before but after doing it I deleted the repository and after a few days I answered other peoples discussions in hopes of getting the badge but no luck, does anyone know when the deleted repository cancels itself out or something like that? I'd like to get this achievement.

**Comment by jge162** [comment]
@S1i Galaxy brain achievement is earned when answering questions in discussions here. The question you answer must be submitted as answered by the user you helped. Like if you click answered question now. it is considered earning towards achievement. But, you need to answer someones question.

**Reply by jge162** [reply]
Also note that you have to answer in specific topics too. See where is says answers enabled. 

<img width="1248" alt="Screen Shot 2023-01-27 at 12 36 09 PM" src="https://user-images.githubusercontent.com/31228460/215191988-fcfcf63b-df42-4243-9ecb-b9247ec6cf36.png">

**Reply by S1i** [reply]
Yes, I have commented on questions in the Q&A section and had my comment marked as the answer multiple times but still no badge ☹
![image](https://user-images.githubusercontent.com/123405324/215200515-ff44f34e-399c-44cd-a3b0-05feb1d2e847.png)

**Reply by jge162** [reply]
Try other topics

**Reply by S1i** [reply]
I tried 2 different topics now and after getting my comment on someone else's question marked as the answer I haven't got the badge

**Reply by jge162** [reply]
You are not answering question in relevant topics. Go to the technical questions not Q&A.

**Reply by jge162** [reply]
> I tried 2 different topics now and after getting my comment on someone else's question marked as the answer I haven't got the badge

You have to actually answer a real question. Not what you are doing now.

**Reply by jge162** [reply]
Hopefully I answered your question today. Bests to you.

**Reply by S1i** [reply]
Thank you

**Reply by jge162** [reply]
If you’re satisfied can u marked answered to close ticket. Thanks.

**Reply by S1i** [reply]
I'll try out your answer and if it works I'll mark your answer as the answer.

**Reply by jge162** [reply]
Perfect.

**Comment by jge162** [comment, marked as answer]
also make sure that you do not have any answers mark by you, for a question you posted. it will prevent you from getting badge @S1i

**Reply by S1i** [reply]
That was almost what the whole body of my question was about

**Comment by shubheshswain91** [comment]
To get the Galaxy Brain achievement badge, you need to fulfill the following criteria:

Discussions should be in a public repository
Get marked as answers to 2 or more questions
The questions cannot be asked by you. In short, you cannot answer your own questions, those do not count.
In fact, answering your own questions could prevent the achievement from displaying in your profile page.

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
